builtin/providers/vsphere/config.go

```diff
@@ -0,0 +1,39 @@
+package vsphere
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+
+	"github.com/vmware/govmomi"
+	"golang.org/x/net/context"
+)
+
+const (
+	defaultInsecureFlag = true
```

Comment:
This is a security hole.
SSL certificate validation must be switched on by default, and has to be disabled by a user explicitly.